src/Makefile.inc

```diff
@@ -213,31 +215,30 @@ $(MODELDIR):
 #The main executable #
 ######################
 $(SDIR)$(TARGET).$(EXEEXT): $(LINKFILE) $(OBJS) $(LIBOPENCM3)
-	@echo " + Building '$@' '$(SRC_CXX)'"
+	@echo " + Building '$(notdir $@)'"
 ifdef LINKFILE #Create an empty 'obj/$(TARGET)/optimize.ld' just in case the linker script needs it
 	echo "" > $(ODIR)optimize.ld
 endif
 ifeq ("$(SRC_CXX)", "")
-	$(CC) -o $@ $(OBJS) $(LIBOPENCM3) $(LFLAGS) $(CFLAGS) $(EXTRA_CFLAGS)
+	$(CC) -o $@ $(OBJS) $(LIBOPENCM3) $(LFLAGS) $(LFLAGS2) $(CFLAGS) $(EXTRA_CFLAGS)
 else
```

Comment:
Can we name his better? I have no idea what the difference is between LFLAGS and LFLAGS2 (Yes I know you didn't create it, but now is a good time to fix it)